crates/router/src/connector/adyen.rs

```diff
@@ -735,6 +735,9 @@ impl api::IncomingWebhook for Adyen {
 
         Ok(match notif.event_code.as_str() {
             "AUTHORISATION" => api::IncomingWebhookEvent::PaymentIntentSuccess,
+            "REFUND" => api::IncomingWebhookEvent::RefundSuccess,
+            "CANCEL_OR_REFUND" => api::IncomingWebhookEvent::RefundSuccess,
+            "REFUND_FAILED" => api::IncomingWebhookEvent::RefundFailure,
```

Comment:
If these event codes have been documented on their docs, please convert this to an enum instead.

Reply:
okay